### PR TITLE
Shorten user-facing error messages

### PR DESCRIPTION
**Pull request by cpotteck**
This is a potential solution for error messages that are too long. I edited them in-place for simplicity. If we want to keep the more descriptive messages for some reason, I could add a layer in `calculate.py` that does the substitution for user-facing messages.

### CODE REVIEW

File: functions/exponents_and_logs.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def pow(base: float, exponent: float, root_used: int = int(1E10)) -> float:
     fractional_part_of_exponent = exponent - integer_part_of_exponent
     # Ensure that base is not negative if exponent is not a integer.
     if common.is_negative(base) and fractional_part_of_exponent != 0:
-        raise exceptions.InputError(None, "Implementation of exponent function for real exponents does not support negative bases because the power function for negative bases is non-continuous over the set of real numbers.")
+        raise exceptions.InputError(None, "negative base with fractional exponent.")
```
Comment (neoJINXD) [inline functions/exponents_and_logs.py:149]: Watch out for capitalization for error message, should probably have them all be similar

File: functions/output_display.py [hunk 2]
```diff
@@ -122,7 +122,7 @@ def binary_to_decimal_integer(value: str) -> int:
     if value[0] == '-':
         raise exceptions.InputError(value, "binary_to_decimal_integer does not accept negative inputs.")
     if not is_binary(value):
-        raise exceptions.InputError(value, "Binary numbers can only contain 1s and 0s.")
+        raise exceptions.InputError(value, "not in binary.")
```
Comment (neoJINXD) [inline functions/output_display.py:125]: Same as the other

File: functions/output_display.py [hunk 3]
```diff
@@ -148,7 +148,7 @@ def binary_to_decimal_fraction(value: str) -> float:
     if value[0] == '-':
         raise exceptions.InputError(value, "binary_to_decimal_fraction does not accept negative inputs.")
     if not is_binary(value):
-        raise exceptions.InputError(value, "Binary numbers can only contain 1s and 0s.")
+        raise exceptions.InputError(value, "not in binary.")
```
Comment (neoJINXD) [inline functions/output_display.py:151]: Same as the other

**Comment by cpotteck** [comment]
will fix. Any comments on what I wrote in the PR message?

**Comment by neoJINXD** [comment]
Yeah, I do believe its a good idea to have an option to flip between simpler and more detailed error messages to make it easier if we ever wanted to switch it or add an option for it somewhere

**Comment by cpotteck** [comment]
> Yeah, I do believe its a good idea to have an option to flip between simpler and more detailed error messages to make it easier if we ever wanted to switch it or add an option for it somewhere

I can add it as a task but since it has no impact on grading I will go with the simplest solution (this one) for now

File: functions/common.py [hunk 4]
```diff
@@ -16,11 +16,11 @@ def factorial(argument: int) -> int:
     """
     # Ensure that input is an integer.
     if int(argument) != argument:
-        raise exceptions.InputError(argument, "Input to factorial should be a non-negative integer.")
+        raise exceptions.InputError(argument, "Input to factorial must be a positive integer.")
```
Comment (EmanuelUnity3D) [inline functions/common.py:19]: I'm a little late to this PR but the idea here was that 0 is not a positive integer. Typically, positive integer refers to all integers greater than 0 and non-negative integer refers to all integers greater than or equal to 0.

**Pull request merged**